_data/cloud.yml

```diff
@@ -46,6 +46,15 @@ websites:
           text: "Currently in beta as of June 5th 2014. Create a support ticket to request 2FA added to your organization"
       doc: https://support.cloud.engineyard.com/entries/40538256-Two-factor-authentication
 
+    - name: fortrabbit
+      url: http://www.fortrabbit.com/
+      twitter: fortrabbit
```

Comment:
@Carlgo11 it's not wrong to have it there. having an extra commit just to remove twitter needlessly pollutes commit log.